test/functional/CommandCallFunctional.js

```diff
@@ -18,11 +18,17 @@ describe('CommandCall Functional Tests', function () {
       connection.add(new CommandCall({ command: 'RTVJOBA USRLIBL(?) SYSLIBL(?)', type: 'cl' }));
       connection.run((error, xmlOut) => {
         expect(error).to.equal(null);
-        parseString(xmlOut, (parseError, result) => {
-          expect(parseError).to.equal(null);
-          expect(result.myscript.cmd[0].success[0]).to.include('+++ success RTVJOBA USRLIBL(?) SYSLIBL(?)');
-          done();
-        });
+
+        const parser = new XMLParser();
+        let result;
+        try {
```

Comment:
@sravani1510 

Looks like we don't need the try/catch blocks here because `.parse` does not return an error. Instead it appears to return an empty object when it is unable to parse the input XML.

I think we should instead have an assertion of `result` and check that the the object keys are > 0.